course_content/notebooks/iris_intro.ipynb

```diff
@@ -502,29 +480,29 @@
     "collapsed": true
    },
    "outputs": [],
-   "source": []
+   "source": [
+    "!ncdump -h saved_cubes.nc | head -n 20\n",
+    "!rm saved_cubes.nc"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "4\\. Print the attributes of the cube."
+    "Extra keywords can be passed to specific fileformat savers."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
-   "source": []
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "5\\. Print the names of all coordinates on the cube. (Hint: Remember the cube.coords method without any keywords will give us all of the cube's coordinates)"
+    "### Deferred (lazy) data\n",
+    "\n",
+    "Deferred data (or lazy data) are terms used to describe functionality provided in Iris that allows large datasets to be loaded as Iris cubes without the dataset's data payload being loaded into memory. As well as lazy data loading, much of the data processing functionality provided by Iris can also operate lazily, and cubes with lazy data can even be saved to disk.\n",
```

Comment:
Is there any way to make this sentence flow a bit better?  There's nothing wrong with it technically, but it's the bit that goes: '...without the dataset's data payload being loaded into memory' that is a bit like a tongue-twister.

I would not look forward to reading that out loud in front of a class...

Reply:
I'll have a go!